python/maps_creator_atlas_data.py

```diff
@@ -70,6 +70,9 @@ def plot(data, project, method,
     Plot relative precipitation and temperature using cartopy.
     """
 
+    # time periods info
+    periods = "projected changes between 2041-2060 mean conditions with respect to the 1995-2014 present-day baseline"
```

Comment:
```suggestion
    periods = "projected changes between 2041-2060 mean conditions with respect to the 1995-2014 baseline"
```

since present day is already a bit beyond 2014

Reply:
yes, I noticed this too, it was written in the feedback and confused me. Thanks for the check.